docs/vercel.json

```diff
@@ -55,6 +55,36 @@
       "destination": "/api/python-api/:path*",
       "permanent": false
     },
+    {
+      "source": "/_apidocs/:path*/",
+      "destination": "/api/python-api/:path*",
+      "permanent": false
+    },
+    {
+      "source": "/sections/api/apidocs/:path*/",
+      "destination": "/api/python-api/:path*",
+      "permanent": false
+    },
+    {
+      "source": "/sections/api/apidocs/:path*",
+      "destination": "/api/python-api/:path*",
+      "permanent": false
+    },
+    {
+      "source": "/docs/apidocs/:path*",
+      "destination": "/api/python-api/:path*",
+      "permanent": false
+    },
+    {
+      "source": "/master/_apidocs/:path*",
+      "destination": "/api/python-api/:path*",
+      "permanent": false
+    },
+    {
+      "source": "/api/:path*",
```

Comment:
Just a random remark - but this prevents us from using `/api/` moving forward, or at least until this is removed, which is totally fine, but just something to be careful of.

Reply:
Oh good call -- we want to keep that path. I'll update.